packages/sit-onyx/src/components/OnyxHeadline/OnyxHeadline.vue

```diff
@@ -0,0 +1,66 @@
+<script lang="ts" setup>
+import type { OnyxHeadlineProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxHeadlineProps>(), {
+  monospace: false,
+});
+
+defineSlots<{
+  /**
+   * Headline content.
+   */
+  default(): unknown;
+}>();
+</script>
+
+<template>
+  <component
+    :is="props.is"
+    :class="[
+      'onyx-headline',
+      `onyx-headline--${props.is}`,
+      props.monospace ? 'onyx-headline--monospace' : '',
+    ]"
+  >
+    <slot></slot>
+  </component>
+</template>
+
+<style lang="scss">
+// use :where for styles that should be able to be easily overridden by the user
```

Comment:
[optional/opinion] sounds a bit raggedly, deepl suggests rephrasing:
```suggestion
// use :where for styles that can be easily overwritten by the user
```